Gemfile

```
@@ -2,13 +2,11 @@
 
 source 'https://rubygems.org'
 
-# 1.12.x and higher, starting from 1.12.1, because that hotfix fixes Xcode 14.3 compatibility
-gem 'cocoapods', '~> 1.12', '>= 1.12.1'
+gem 'cocoapods', '~> 1.14'
 gem 'commonmarker'
-gem 'danger', '~> 9.3'
-gem 'danger-rubocop', '~> 0.10'
+gem 'danger-dangermattic', git: 'https://github.com/Automattic/dangermattic'
 gem 'dotenv'
-gem 'fastlane', '~> 2.174'
+gem 'fastlane', '~> 2'
```

There have been cases in which we had to update Fastlane, e.g. 6c989713647a70fa3731ccc8f6efa56e52764704 , to get a version that fixed an issue. I worry that leaving it open to `~> 2` _might_ result in accidentally going back to an older version if other gems require it (I admit the case is remote).

What do you think of using 

```ruby
gem 'fastlane', '~> 2.<latest version>'
```

or, in case we need a hotfix

```ruby
gem 'fastlane', '~> 2.<latest minor version>', '>= 2.<latest minor version>.<patch>'
```

`bundle show fastlane` gives me version 2.216.0, so:

```suggestion
gem 'fastlane', '~> 2.216'
```